CHANGELOG.md

```diff
@@ -22,6 +22,22 @@ We'd like to offer a big thanks to the 7 contributors who made this release poss
 - 📝 Add `onProcessRowUpdateError` prop to simplify error management in edit mode (#4267) @m4theushw
 - ✨  Add generic typing to `GridColDef` and derived interfaces (#4064) @flaviendelangle
 
+  This allows to strongly type all the objects related to the row and the cell values.
+  Here is an example, you can can find out more in the description of #4064.
```

Comment:
I don't know if we should link to the description of a PR
Maybe for now it's the best solution, but we can create an issue to do a "TypeScript" data grid doc page and change the CHANGELOG to link to this new page when available.

Reply:
done -> https://github.com/mui/mui-x/issues/4330